SingularityUI/app/reducers/api/base.es6

```diff
@@ -13,7 +13,9 @@ export default function buildApiActionReducer(ActionGroup, initialData=[]) {
       case ActionGroup.ERROR:
         return _.extend({}, state, {
           isFetching: false,
-          error: action.error
+          error: action.error,
+          data: null,
```

Comment:
Actually, I don't even need to do that - I thought I did before I got deep copying right. I'll remove it entirely.